src/Okta.Sdk.IntegrationTests/TrustedOriginScenarios.cs

```diff
@@ -0,0 +1,241 @@
+﻿// <copyright file="TrustedOriginScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class TrustedOriginScenarios
+    {
+        private static string dotnetSdkPrefix = "dotnetSdk";
+
+        [Fact]
+        public async Task ListOrigins()
+        {
+            var client = TestClient.Create();
+            var trustedOrigins = await client.TrustedOrigins.ListOrigins().ToListAsync();
```

Comment:
To test this scenario, let's create a Trusted Origin and verify if it is listed by this method. We shouldn't rely on existing origins.